libs/ui-lib/lib/ocm/hooks/use-feature-detection.ts

```diff
@@ -15,3 +20,7 @@ export function useFeatureDetection<T extends Partial<Record<string, boolean> &
     );
   }, [overrides]);
 }
+
+export const useFeature = (feature: AssistedInstallerFeatureType): boolean => {
```

Comment:
Please move this function into a different file called `use-feature`. It makes no sense to have it defined in a module called  `use-feature-detection`

Reply:
done